source/_components/homekit.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: "HomeKit"
+description: "Instructions how to setup the HomeKit component in Home Assistant."
+date: 2018-02-20 17:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: HomeKit
+ha_release: 0.64
+ha_iot_class:  "Local Polling"
+logo: home-assistant.png
+---
+
+The `HomeKit` component allows you to forward entities from Home Assistant to `HomeKit`, so they could be controlled from their as well.
+
+{% configuration %}
+  homekit:
+    description: HomeKit configuration.
+    required: true
+    type: map
+    keys:
+      pincode:
+        description: Pincode required during setup of HomeKit Home Assistant accessory. The Format needs to be 'XXX-XX-XXX' where X is a number between 1 and 9.
+        required: false
+        type: string
+        default: '"123-45-678"'
+      port:
+        description: Port for the HomeKit extension.
+        required: false
+        type: int
+        default: 51826
+{% endconfiguration %}
+
+## {% linkable_title Setup %}
+
+To enable the `HomeKit` component in Home Assistant, add the following to your configurations file:
+
+```yaml
+# Example for HomeKit setup
+homekit:
+  pincode: '123-45-678'
+```
+
+<p class='note'>It is not recommended to choose '123-45-678' as your pincode! The Pincode must be of the format 'XXX-XX-XXX' where X is a number between 1 and 9.</p>
+
+After Home Assistant has started, all supported entities (see the [list](#supported-components) belove which entities are already) will be ready to be added to `HomeKit`. To add them:
+1. Open the `HomeKit` App.
+2. Choose `Add Accessory` and then `Don't Have a Code or Can't Scan?`.
```

Comment:
sentence style for capitalization?

Reply:
`Don't Have a Code or Can't Scan?` that's how it's written in the app.

Reply:
so keep it as it's in the app :)